docs/user-guide/import-certificates.md

```diff
@@ -1,13 +1,172 @@
-# Configure certificates if you have existing certificates
+# Import and configure an existing certificate 
 
-## Import your PKCS12 certificate
+Zowe makes it possible to import and configure existing certificates. The following certificates are supported by Zowe:
+* PKCS12 certificates
+* JCERACFKS certificates 
+
+Review the procedure to import existing PKCS12 or JCERACFKS certificates as well as how to import a certificate stored in an MVS data set into a Zowe keyring.
+## Import an existing PKCS12 certificate
+
+To import a PKCS12 certificate, it is first necessary to import a certificate authority. Then, you can configure the section `>>>> Certificate setup scenario 2` in `zowe.yaml`. 
+
+For PKCS12 certificate users, specify the following parameters in the `zowe.yaml` file:
+
+* Define `zowe.setup.certificate.pkcs12.import.keystore` if you acquired one or more certificates from another CA, stored them in PKCS12 format, and now want to import the certificate(s) into the Zowe PKCS12 keystore.
+
+* Specify `zowe.setup.certificate.pkcs12.import.password`  
+ This value is the password for keystore defined in `zowe.setup.certificate.pkcs12.import.keystore`.
+
+* Specify `zowe.setup.certificate.pkcs12.import.alias`  
+This value is the original certificate alias defined in `zowe.setup.certificate.pkcs12.import.keystore`.
+
+    The imported certificate is saved under the alias specified in `zowe.setup.certificate.pkcs12.name`.
+
+**Example YAML:**
+```
+zowe:
+  setup:
+    certificate:
+      type: PKCS12
+      pkcs12:
+        directory: /var/zowe/keystore
+        lock: true
+          name: localhost
+          password: password
+        import:
+          keystore: ""
+          password: ""
+          alias: ""
+      importCertificateAuthorities:
+        - ""
+```
+
+**Note:**  
+Due to the limitation of RACDCERT command, the `importCertificateAuthorities` field can contain a maximum of two entries.
 
-Before you import a PKCS12 certificate, you need to import a certificate authority first.
 
 ### Manually import a certificate authority into a web browser
 
-To import an existing certificate and avoid the browser untrusted CA challenge, you have to import Zowe's certificates into the browser to avoid untrusted network traffic challenges. For more information, see [Import the local CA certificate to your browser](../extend/extend-apiml/certificate-management-in-zowe-apiml.md/#import-the-local-ca-certificate-to-your-browser).
+To import an existing certificate and avoid the browser untrusted CA challenge, import Zowe's certificates into the browser.
```

Comment:
> It seems we should add the procedure or a link to import certificates into a browser.

Good point! But it might depend on what browser you use. I will make a comparison with different browsers.

Reply:
I agree, good UX to provide link to fix the issue. Would be a good standard throughout Zowe (if it isn't already), we are finding out that a chunk of the audience is in unfamiliar territory.